### ISSUE TITLE
--open don't work for me

### ISSUE THREAD

**Issue by cebor** [webpack/webpack-dev-server#378]
just tested it. `--open` does not open a browser window on osx

**Comment by ImaginateWayne** [comment]
I have the same issue, also on osx

**Comment by ImaginateWayne** [comment]
@cebor what version are you on? After looking through the code it appears that version 1.4.1 doesn't have the `--open` flag this is a 2.x.x feature.

**Comment by cebor** [comment]
```json
{
    "webpack": "^1.12.12",
    "webpack-dev-server": "^1.14.1"
}
```

hmm ok

**Comment by katopz** [comment]
@cebor Get it working? which version btw? All I get is `UNMET PEER DEPENDENCY` when try to use 2.0.0 beta both `webpack` and `web-pack-dev-server`

**Comment by cebor** [comment]
@katopz i haven't tried it with 2.0.0-beta yet, waiting for final

**Comment by alexilyaev** [comment]
@ImaginateWayne Can we add a note in the documentation for features that are only available from 2.0.0?
https://webpack.github.io/docs/webpack-dev-server.html#webpack-dev-server-cli

**Comment by ImaginateWayne** [comment]
It has been updated!
